dts/common/yaml/device_node.yaml.template

```diff
@@ -13,10 +13,11 @@ properties:
 
 # A typical property entry will look like the following
 # - <name of property as it is in device tree>
-#     category: <required | optional>
+#     category: <required | required-implicit | optional>
 #     type: <string | int | array>
 #     description: <description of property>
 #     generation: <define | structure>
+#     default-value: keyword for required-implicit generation
```

Comment:
what values do we support for 'default-value'?

Reply:
Foe now: parent-label